src/pages/tokens/[id].js

```diff
@@ -172,41 +203,57 @@ function TokenPage() {
     <AppShell>
       <Head>
         <title>
-          {currencyFormatter.format(price || 0)} | {token.symbol} | Trader Joe
+          {currencyFormatter.format(price || 0)} | {token.symbol} |{" "}
+          {process.env.NEXT_PUBLIC_APP_NAME}
           Analytics
         </title>
       </Head>
       <PageHeader>
+        <AutoRow align="flex-end" pb={3} style={{ width: 'fit-content' }}>
+          <TYPE.body>
+            <NextLink href="/tokens">{'Tokens '}</NextLink>→ {token.symbol}
+            {'  '}
+          </TYPE.body>
+          <Link
+            style={{ width: 'fit-content', color: "rgb(120, 134, 134)" }}
+            external
+            href={'https://explorer.fuse.io/address/' + id}
+          >
+            <Typography style={{ marginLeft: '.15rem' }} fontSize={'14px'} fontWeight={400}>
+              ({token.id.slice(0, 8) + '...' + token.id.slice(36, 42)})
+            </Typography>
+          </Link>
+        </AutoRow>
         <Grid
           container
           direction="row"
           justify="space-between"
           alignItems="center"
         >
           <Grid item xs={12} sm="auto" className={classes.title}>
-            <Box display="flex" alignItems="center">
-              <TokenIcon id={token.id} />
-              <Typography variant="h5" component="h1" noWrap>
+            <Box display="flex" alignItems="center" mt={2}>
+              <TokenIcon id={token.symbol} width="40px" height="40px" />
+              <Typography variant="h4" component="h1" noWrap>
                 {token.name} ({token.symbol}){" "}
               </Typography>
             </Box>
-            <Box display="flex" alignItems="center" className={classes.price}>
-              <Typography variant="h6" component="div">
+            <Box display="flex" alignItems="flex-end" className={classes.price}>
+              <Typography variant="h5" component="div">
                 {currencyFormatter.format(price || 0)}
               </Typography>
               <Percent percent={priceChange} ml={1} />
             </Box>
           </Grid>
           <Grid item xs={12} sm="auto" className={classes.links}>
             <Link
-              href={`https://traderjoexyz.com/pool/${token.id}/AVAX`}
+              href={`https://app.voltage.finance/#/add/${token.id}/FUSE`}
               target="_blank"
               variant="body1"
             >
               Add Liquidity
             </Link>
             <Link
-              href={`https://traderjoexyz.com/trade`}
```

Comment:
url should be https://app.voltage.com/swap

Reply:
✅